docs/source/advanced/boosted_frame.rst

```diff
@@ -184,3 +197,136 @@ class to a velocity close to:
+
+Here :math:`v_{window} = \beta_{window}c` is speed of the moving window (which is typically
+set to the phase velocity of the plasma wake :math:`\beta_{window} = \beta_{wake}`)
+and :math:`v_{plasma} = \beta_{plasma}c` is the speed of the plasma, which would
+be :math:`v_{plasma} = 0` in the lab frame.
+
+Consequently, the number of iterations in the lab frame are given by:
+
+.. math::
+  N_{lab} = \frac{T_{interact}}{\Delta t}
+
+When using the boosted frame technique, FBPIC will transform all those
+quantities into the new frame of reference:
+
+.. math::
+  L_{plasma}' = \frac{L_{plasma}}{\gamma}
+.. math::
+  l_{window}' = l_{window}\gamma(1+\beta)
+.. math::
+  v_{window}' = c( (\beta_{window}-\beta)/(1-\beta_{window}\beta) )
+.. math::
+  v_{plasma}' = -\beta c
+
+and using the above formula we can calculate the interaction time in the
+boosted frame. The number of required iterations are then given by:
+
+.. math::
+  N_{boost} = \frac{T_{interact}'}{\Delta t'}
+
+where :math:`\Delta t' = \gamma (1+\beta) \Delta t` is
+the timestep in the boosted frame.
+
+The :any:`BoostConverter` object exposes the above formula for :math:`T_{interact}'`
+as function :any:`interaction_time` that can be used to calculate the
+required number of timesteps in the boosted frame for a given moving window sliding
+across an initially static object with length :math:`L_{interact}` (e.g.
+the plasma :math:`L_{interact}=L_{plasma}`). In addition, the timestep in
+the boosted frame :math:`\Delta t'` can be accessed through ``sim.dt`` in
+FBPIC (see the boosted-frame script in :doc:`../how_to_run`).
+
+The following figure shows the calculation of the expected speedup
+(defined as the reduction in the number of PIC iterations, :math:`N_{lab}/N_{boost}`)
+of a typical laser-plasma acceleration case. Lets assume a simulation box
+(moving window) of length :math:`l_{window} = 100 \, \mu m` interacting with a
+plasma of length :math:`L_{plasma} = 12 \, mm`. Assuming a plasma density of
+:math:`n_{e}=10^{24} \, m^{-3}`, we set the velocity of the moving window to match
+the plasma wake phase velocity, i.e. :math:`\gamma_{wake} \approx 42`.
+
+.. image:: ../images/boosted_frame_speedup_example.svg
+
+This simple example highlights two important aspects of choosing the right
+:math:`\gamma_b` in practice. First, it can be seen that the speedup does
+only follow the simple scaling law :math:`(1+\beta_b^2)\gamma_b^2`
+for :math:`\gamma_b \ll \gamma_{wake}`, and second, that the optimum speedup
+occurs at a :math:`\gamma_b < \gamma_{wake}` before the simulation
+slows down again.
+
+.. note::
+
+    As highlighted in the above example, there is an optimum
+    :math:`\gamma_b` for which the interaction time is minimized for a
+    given simulation setup. The speed up of a boosted frame
+    simulation will reverse for too high values of :math:`\gamma_b`.
+    In practice, it is often advisable to choose :math:`\gamma_b < \gamma_{wake}/2`
```

Comment:
Replace with new formulation: 

In practice, it is often advisable to fulfil :math:`\gamma_b^2 < L_{plasma}/l_{window}` for maximum efficiency and :math:`\gamma_b < \gamma_{wake}/2` to properly resolve the physical objects (see next section).